{{cookiecutter.project_shortname}}/Dockerfile.dev.base

```diff
@@ -0,0 +1,9 @@
+{% include 'misc/header.py' %}
+FROM inveniosoftware/centos7-python:3.6
+
+COPY Pipfile Pipfile.lock ./
+COPY ./docker/uwsgi/ ${INSTANCE_PATH}
+
+USER ${USER_ID}
+
+RUN pipenv install --deploy --dev --pre
```

Comment:
The `--pre` and `--dev` flags are moved here as they are meant for a dev environment. #87